plugins/woocommerce-blocks/assets/js/blocks/product-filters/frontend.ts

```diff
@@ -99,9 +69,10 @@ store( 'woocommerce/product-filters', {
 		},
 	},
 	callbacks: {
-		maybeNavigate: () => {
+		*maybeNavigate() {
 			const { params, originalParams } =
 				getContext< ProductFiltersContext >();
+			console.log( params, originalParams );
 
 			if ( isParamsEqual( params, originalParams ) ) {
 				return;
```

Comment:
@DAreRodz these two contexts are updated/merged with new ones from the navigation, it did previously.

Reply:
In the render callback of the Product Filters block, I set the same value for `params` and `originalParams`, which is current URL params, so after navigation, the value of them should be the same. But they're not updated after navigation with the new API

```
			$tags->set_attribute(
				'data-wc-context',
				wp_json_encode(
					array(
						'isDialogOpen'                 => false,
						'hasPageWithWordPressAdminBar' => false,
						'params'                       => $this->get_filter_query_params( 0 ),
						'originalParams'               => $this->get_filter_query_params( 0 ),
					),
					JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_QUOT | JSON_HEX_AMP
				)
			);
```